operator/test/test.sh

```diff
@@ -4,7 +4,6 @@
 set -euo pipefail
 
 # Uncomment the below line to enable debugging
-# set -x
```

Comment:
This line should be kept (or the comment above should be removed as well).

Reply:
Fixed